LibClassicSwingTimerAPI.toc

```diff
@@ -1,8 +1,8 @@
-## Interface: 100200
+## Interface: 100200,40400
```

Comment:
I think we can add a specific line ## Interface-Cata: 40400 no ?

Reply:
You're correct! Also I didn't mean to click resolve